packages/translate/translations.json

```diff
@@ -1694,6 +1694,18 @@
       "key": "api/publish/disabled",
       "value": "Artikel können im Moment weder veröffentlicht noch zurückgezogen werden. Bitte versuche es in ein paar Minuten noch einmal, ansonsten kontaktiere bitte die IT."
     },
+    {
+      "key": "api/publish/error/createCampaign",
+      "value": "Kampagne auf MailChimp konnte nicht angelegt werden."
+    },
+    {
+      "key": "api/publish/error/updateCampaign",
+      "value": "Kampagne auf MailChimp konnte nicht aktualisiert werden."
+    },
+    {
+      "key": "api/publish/error/updateCampaignContent",
+      "value": "Kampagne auf MailChimp konnte nicht aktualisiert werden."
```

Comment:
```suggestion
      "value": "Kampagne auf MailChimp konnte nicht aktualisiert werden. (updateCampaignContent)"
```

Reply:
This error message (and `updateCampaign`) are intended for regular users, indicating the realm of the error (campaign update failed). While this is posted via API, [we will log an error with stack trace](https://github.com/orbiting/backends/pull/436/files#diff-6e358742c5bc46972ccb65ea7e02ad7fR478) and enabling us to debug.